boards/nrf52840dongle/Makefile.features

```diff
@@ -0,0 +1,10 @@
+CPU_MODEL = nrf52840xxaa
+
+# Put defined MCU peripherals here (in alphabetical order)
+FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_usbdev
+
+# Various other features (if any)
+FEATURES_PROVIDED += radio_nrf802154
+
+include $(RIOTBOARD)/common/nrf52/Makefile.features
```

Comment:
```suggestion
FEATURES_PROVIDED += periph_rtt
FEATURES_PROVIDED += periph_timer
```

That should be enough so that this doesn't get the `riotboot` feature.

Reply:
Shouldn't that error rather be fixed than introducing a lot of code duplication?

Reply:
Hm, I have a better idea

```patch
--- a/boards/common/nrf52/Makefile.features
+++ b/boards/common/nrf52/Makefile.features
@@ -3,7 +3,7 @@ CPU = nrf52
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_rtt
 FEATURES_PROVIDED += periph_timer
-ifeq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+ifeq (,$(filter nordic_softdevice_ble bootloader_nrfutil,$(USEPKG)))
   FEATURES_PROVIDED += riotboot
 endif
```

Reply:
bootloader_nrfutil is a feature not a package; adapted accordingly and setting bootloader_nrfutil for the board (it was not previously) in 981c1411d4dff28848f4ce85d2306ec89b5396d5